controllers/teleport_master_controller/config.yaml

```
@@ -2,46 +2,46 @@
 # gain a range of angles
 rotations:
   -
-    - -0.10413
-    - 0.0830422
-    - 0.991091
-    - 1.53982
-  -
-    - -0.0489131
-    - 0.0885032
-    - 0.994874
-    - 2.32004
-  -
-    - -0.0103814
-    - 0.0921435
-    - 0.995692
-    - 3.10184
-  -
-    - 0.0275397
-    - 0.0955899
-    - 0.99504
-    - -2.39949
-  -
-    - -0.0798456
-    - -0.100122
-    - -0.991766
-    - 1.61898
-  -
-    - -0.197932
-    - -0.109392
-    - -0.974093
-    - 0.84478
-  -
-    - -0.971637
-    - -0.109473
-    - -0.209614
-    - 0.189914
-  -
-    - -0.238027
-    - 0.0685756
-    - 0.968835
-    - 0.767201
+    - 0
+    - 1
+    - 0
+    - 0
+  -
+    - 0
+    - 0
+    - 1
+    - 0.785398
+  -
+    - 0
+    - 0
+    - 1
+    - 1.5708
+  -
+    - 0
+    - 0
+    - 1
+    - 2.3562
+  -
+    - 0
+    - 0
+    - 1
+    - -3.14159
+  -
+    - 0
+    - 0
+    - -1
+    - 2.3562
+  -
+    - 0
+    - 0
+    - -1
+    - 1.5708
+  -
+    - 0
+    - 0
+    - -1
+    - 0.785398
 
-minDistance: 0.1
+minDistance: 0.5
```

What is this? min distance between what? comment please
```suggestion
min_distance: 0.5
```